CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Version 0.8.0 (UNRELEASED)
+---------------------------
+
+- Changes base image to the official RabbitMQ Docker image
+- Adds RabbitMQ message delay plugin
```

Comment:
Perhaps add a line about management interface exposed on port XYZ in the cluster debug mode.

Reply:
Mentioned `31672` port which we expose in Helm, even though it's `15672` inside the container. I guess it's more useful to know the one to access the management UI